client/src/features/home/NeedsAndOffers.tsx

```diff
@@ -65,9 +65,11 @@ function NeedsAndOffers(props: Props): JSX.Element {
         {assets.map((asset) => (
           <NavLink to={`/asset/${asset.id}`} key={asset.id} className={classes.card}>
             <Card variant="outlined">
-              <img src={asset.imgUrls[0]} className={classes.cardImg} alt={asset.title} />
+              {asset.imgUrls ? (
+                <img src={asset.imgUrls[0]} className={classes.cardImg} alt={asset.title} />
+              ) : null}
```

Comment:
hmm, maybe i take it back....
go with the default image solution you had before